.github/workflows/sourcehold.yml

```diff
@@ -17,47 +17,90 @@ jobs:
     strategy:
       matrix:
         os: [macos-latest, ubuntu-latest]
+        target-platform: ["host"]
         include:
           - os: ubuntu-latest
-            install_dependencies: "sh ./ubuntu/install-dependencies.sh"
-            artifact_name: "sourcehold-linux-amd64"
+            install-dependencies: "sh ./ubuntu/install-dependencies.sh"
+            cmake-configure-options: ""
+            cmake-build-options: ""
+            uploaded-artifact-name: "sourcehold-linux-amd64"
           - os: macos-latest
-            install_dependencies: "sh ./macos/install-dependencies.sh"
-            artifact_name: "sourcehold-mac-os"
+            install-dependencies: "sh ./apple/install-dependencies-macos.sh"
+            cmake-configure-options: ""
+            cmake-build-options: ""
+            uploaded-artifact-name: "sourcehold-mac-os"
+          # Use different target platform here just to force new job spawning
+          - target-platform: "ios-simulator"
+            os: macos-latest
+            install-dependencies: "sh ./apple/install-dependencies-ios.sh -s -d 11.0 ./thirdparty/ios"
+            env:
+              XCODE_PATH="$(xcode-select -p)"
+              SIMULATOR_SYSROOT_PATH="$XCODE_PATH/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator.sdk"
+            cmake-configure-options: '-GXcode -DCMAKE_SYSTEM_NAME=iOS -DCMAKE_OSX_DEPLOYMENT_TARGET=11.0 -DCMAKE_APPLE_ARCH_SYSROOTS="$SYSROOT_PATH" -DCMAKE_OSX_ARCHITECTURES="x86_64"'
+            cmake-build-options: "-- -sdk iphonesimulator"
+            uploaded-artifact-name: "sourcehold-ios"
+    
+    name: ${{ matrix.os }} - ${{ matrix.target-platform }}
     
     steps:
-    - uses: actions/checkout@v2
+    - name: Set up environment variables
+      # Unfortunately workflow level env variables can not be accessed at job level
+      # (see https://github.community/t/how-to-set-and-access-a-workflow-variable/17335)
+      # in our case this is BUILD_TYPE. Thus we have to define it here in such weird way.
+      run: |
+          echo "CMKAE VERSION = `cmake --version`"
```

Comment:
typo 'CMAKE VERSION'

Reply:
This is test output. I forgot to remove it.